### PR TITLE
Update gitian keys, script and doc

### PR DESCRIPTION
**Pull request by codablock**
See individual commits

### CODE REVIEW

**Comment by PaulieD** [comment]
What's the motivation behind this, what will this allow you to do? Thanks.

**Comment by codablock** [comment]
@PaulieD It never made sense to have the Gitian keys of Bitcoin developers inside the Dash repo. Also, we're currently considering changing the way we do releases (introduce proper Gitian signing by our devs).

File: contrib/gitian-keys/nmarley.pgp [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+0WW4x9HMJ88o0xaw87dEGiifVsffu0sxUGkLpLK6IfMg8EOQGf4GmQYrmhBUZPTE
+2pZXIf8IMbU23R4947r2f9dl2SfIEW3foF5dfTk7F6yzF4STJTqKVyqOdwdPYlOr
+0H1g6+kAPsyWoGdIbohXw1WRp9CJBiPuEVQZ4jHk9/AIzGfx4vlvQWc1pB+2Bmcv
+uvbahL1aW6TQKlM8NoHQUrEJF0Xnm02EJHkqczIyL1OZD2ylJBfGUuKuYvQ3uejV
+6U8b4yj6l/5EngJszjuR8aS8vtl6AGtt+RhJCTfZO8v4R+rPFY0sVJQ8VJyCkrSu
+dr490RWMkBIaL23olNUgEdx/UfQSn3lC89FUNTfZQ34MYa/ORyIMQebzlJZama0n
+IWdiYNzA1p3M3QNrTfMvRCTl80LMQBLhfhzQkDM9JldjGUzMkWpX8awJLEYNmOL2
+On0zI0oMYOhBYYNELJM+4GPqNBWH8NjwZ8/ujz6H30ISyh4KwP0oh4bnvVw=
+=X22E
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (nmarley) [inline contrib/gitian-keys/nmarley.pgp]: Can you add the EOL char here?

File: contrib/gitian-keys/schinzelh.pgp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+/m9QctBtUnSezouFFdjmoQARAQABiQE8BBgBCgAmAhsMFiEErxrhPzPQb0h/I9yB
+S4gmmr2N8zIFAlpXfvsFCQoi9v0ACgkQS4gmmr2N8zJiBAgAu2tgY+H1tEkFJs9r
+D+rdKXDCVy/5Jpogkw9O0FeYWXcQw+ru7GpqRk+fE2a0ETr8RAADFBQC1BeqWnzb
+Y2EpRddobg2Drjs8mlPkN4E+leybxtHIRXl591oLPPDO/yqzjE40f0IxtMnPnKhy
+tNwJpYXv5ZYE6G3+QYOYIGfL4JRXpyHgfS5T7UasEPPXf5Nd3INrEMNEOQw41Jif
+H29pV26nmwc31q/3I9MhuYiM7rn+5zBNDVyhNrmawOqSq/R1iNSxeSsLzecwaA6f
+mhR5sfgrZHXchFBrPv3Gqzy8CZV4N66RTDTObxEb5R5bMt+NMO713f6quEOMStbb
+R/tLzA==
+=CeyW
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (nmarley) [inline contrib/gitian-keys/schinzelh.pgp]: Needs EOL char here.

**Comment by nmarley** [comment]
utACK 0ca36bb81bfc24103209b8e921762ef0172fbbe6 w/EOL char added,

couple small nits (files without terminating newline drive me nuts, and this is a POSIX standard)

**Comment by codablock** [comment]
@PaulieD Our usual release process involves Gitian binaries built with Bamboo, which are then used for tests on testnet. Bamboo is down however and we're struggling making it run again, so we must consider alternatives to using Bamboo for this. Not just for now, but for the future as well. Establishing new keys from Dash developers is one step in this, the next step will be to either manually create signed release binaries and/or setup a new CI environment. We are doing this kind of in parallel now and whatever is finished first might be used for the release.

**Comment by PaulieD** [comment]
Thank you @codablock. For future problems which delay launch, and even this one, I would really appreciate a public update, nothing fancy something just like "We are having a problem with our traditional release flow and are trying to improve it for the future. We hope to have it fixed in a week or two. Sorry for the delay." 

Something like that would really make the community feel better about any delays based on my talking with people. Thanks for the great work, though!

**Pull request merged**